framework/src/play/mvc/Router.java

```diff
@@ -85,18 +85,21 @@ public static void clearForTests() {
         instance.setRoutes(emptyList());
     }
 
-    public static void resetForTests(List<Route> routes) {
-        instance.setRoutes(routes);
-    }
-
     /**
-     * Parse the routes file. This is called at startup.
+     * Parse the routes file and load the routing definitions. This is called at normal startup.
      */
     private static void load() {
-        instance.setRoutes(new RoutesParser().parse(Play.routes));
+        load(new RoutesParser().parse(Play.routes));
         lastLoading = System.currentTimeMillis();
     }
 
+    /**
+     * Load the routing definitions.
+     */
+    public static void load(List<Route> routes) {
```

Comment:
@cies  I would name this method `init`, `set` or `setRoutes`, not `load`. It doesn't really load anything.

Reply:
Good point. Better this way?